src/test/java/com/mongodb/hibernate/jdbc/MongoPreparedStatementTests.java

```diff
@@ -110,33 +120,33 @@ void testSuccess() throws SQLException {
 
             try (var preparedStatement = createMongoPreparedStatement(EXAMPLE_MQL)) {
 
-                preparedStatement.setString(1, "War and Peace");
-                preparedStatement.setString(2, "Leo Tolstoy");
-                preparedStatement.setInt(3, 1869);
-                preparedStatement.setBoolean(4, false);
-                preparedStatement.setString(5, "classic");
+                preparedStatement.setString(1, "s1");
+                preparedStatement.setString(2, "s2");
+                preparedStatement.setInt(3, 1);
+                preparedStatement.setBoolean(4, true);
+                preparedStatement.setString(5, "array element");
+                preparedStatement.setObject(6, new ObjectId(1, 2), MqlType.OBJECT_ID.getVendorTypeNumber());
+                preparedStatement.setObject(7, new ObjectId(2, 0), MqlType.OBJECT_ID.getVendorTypeNumber());
 
                 preparedStatement.executeUpdate();
 
                 verify(mongoDatabase).runCommand(eq(clientSession), commandCaptor.capture());
                 var command = commandCaptor.getValue();
-                var expectedDoc = BsonDocument.parse(
-                        """
-                        {
-                            insert: "books",
-                            documents: [
-                                {
-                                    title: "War and Peace",
-                                    author: "Leo Tolstoy",
-                                    publishYear: 1869,
-                                    outOfStock: false,
-                                    tags: [
-                                        "classic"
-                                    ]
-                                }
-                            ]
-                        }
-                        """);
+                var expectedDoc = new BsonDocument()
+                        .append("insert", new BsonString("items"))
+                        .append(
+                                "documents",
+                                new BsonArray(List.of(new BsonDocument()
+                                        .append("string1", new BsonString("s1"))
+                                        .append("string2", new BsonString("s2"))
+                                        .append("int32", new BsonInt32(1))
+                                        .append("boolean", BsonBoolean.TRUE)
+                                        .append(
+                                                "stringAndObjectId",
+                                                new BsonArray(List.of(
+                                                        new BsonString("array element"),
+                                                        new BsonObjectId(new ObjectId(1, 2)))))
+                                        .append("objectId", new BsonObjectId(new ObjectId(2, 0))))));
```

Comment:
This way it is easier to express what exactly we are expecting than with a string.